### DISCUSSION TITLE
How can I access pyscipopt.scip.Model after wrapping the model by core.scip.Model.from_pyscipopt?

### DISCUSSION THREAD

**Discussion by Wenbo11** [ds4dm/ecole#86] [Questions] [answered]
## Describe the problem or improvement suggested
It might be a dumb question. How can I access pyscipopt.scip.Model after wrapping the model by core.scip.Model.from_pyscipopt?

## Describe the solution you would like
No idea so far.

## Additional context
Enable the methods in pyscipopt for core.scip.Model.

**Comment by jdumouchelle** [comment, marked as answer]
You can use the method `as_pyscipopt()`.  For example:

`pyscipopt_model = ecole_model.as_pyscipopt()`

**Comment by Wenbo11** [comment]
It works. Thank you so much!
